### DISCUSSION TITLE
right side bar on github is not responsive

### DISCUSSION THREAD

**Discussion by EsraaAboudaif** [community/community#55665] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

in home page for github website, the right sidebar is not responsive and gets in the way of the main body

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @esraa00 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by lil5** [comment]
![Screenshot 2023-05-25 at 09 19 16](https://github.com/community/community/assets/17646836/79916adc-1f06-4f29-8286-2ddf5a62b2ae)

@queenofcorgis 

screen width: 1242px
browser: Brave
notable extension: Ghostery

bug: See **Latest changes**
